### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by N0wan** [atorch/probability_puzzles#5]
Hey there !

Would you be interested by a translation in French language ?

**Comment by N0wan** [comment]
I can easily translate all the strings.
But would you provide a multi-language app (needing the code to be adapted) ?
The other solution being to provide one app per language (less good).

**Comment by atorch** [comment]
That would be wonderful! I need to read up on how to make a single app with multiple languages, but that sounds by far like the better option (relative to multiple single-language apps).

**Comment by N0wan** [comment]
Yes indeed.
So, when you'll modify the source code, or add new features, you'll only have to do it once.
Actually that will be my first contribution to a multi-language app... sounds interesting !
Btw, I'm a fan of your app !

**Comment by rillig** [comment]
See https://github.com/atorch/probability_puzzles/pull/7, where I am adding the German translation. The main part is really only adding a file `values-de/strings.xml` containing the translated strings.

Some of the images need to be translated as well as they contain text. I left that for later, but I still have the list for translating the images in my e-mail archive.

**Comment by N0wan** [comment]
For sure, I can do that.
But just to be sure, why can't I find your folder values-de in any branch ?

**Comment by rillig** [comment]
The German translation is in my fork over at https://github.com/rillig/probability_puzzles, in the master branch.
Before starting to translate, you might ask @atorch whether he still wants to release a new version of the app including the translations, just to make sure you don't waste time doing the translation for nothing.

**Comment by N0wan** [comment]
Thanks, I'll do so

**Comment by N0wan** [comment]
Actually, I don't know how to contact him... ?

**Comment by atorch** [comment]
Hey, I'm here!  I see https://github.com/atorch/probability_puzzles/pull/7, I'll give it a try on my phone when I have some free time

**Comment by atorch** [comment]
@N0wan you should now see https://github.com/atorch/probability_puzzles/blob/master/app/src/main/res/values/strings.xml and https://github.com/atorch/probability_puzzles/blob/master/app/src/main/res/values-de/strings.xml

**Comment by esodigoyo** [comment]
Hello, I could do Greek and Spanish translations.

**Comment by atorch** [comment]
@esodigoyo that would be great, thank you for offering! Have a look at https://github.com/atorch/probability_puzzles/issues/20#issuecomment-1199784463 and https://github.com/atorch/probability_puzzles/blob/master/app/src/main/res/values-de/strings.xml and https://developer.android.com/guide/topics/resources/localization#creating-alternatives to see how translations work. Let me know if you need additional tips! This issue is about a French translation specifically, so let's open a separate issue for Greek and Spanish in the event that you have any questions specific to creating a values.xml file for those languages.
